Bottom: cc729bf4ab8ac1a8b549644530ca48162d988d8e
Top:    865c4a3d15f16335a9f78016b231eb60ed8aaef0
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2012-11-24 18:58:20 +0000

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
